featuretools/variable_types/variable.py

```diff
@@ -257,6 +257,24 @@ class EmailAddress(Variable):
     _default_pandas_dtype = str
 
 
+class URL(Variable):
+    """Represents a valid web url"""
```

Comment:
We should put more information in this docstring. Saying that having a url with/without `http` and `www` is okay.